rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnNeverTypeRector/Fixture/require_existing_final_void.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnNeverTypeRector\Fixture;
+
+use Rector\Core\Exception\ShouldNotHappenException;
+
+final class RequireExistingFinalVoid
+{
+    public function run(): void
+    {
+        throw new ShouldNotHappenException('implement by child');
```

Comment:
class is final already, the "implement by child" message seems ambiguos meaning, just echo should be ok for final class

Reply:
Indeed, I'll update text